### ISSUE TITLE
RFC: Standardizing on Github milestones across repos

### ISSUE THREAD

**Issue by antoviaque** [openedx/wg-coordination#121]
By releases, in particular.

* Redwood.1, 2024-06-10
* Redwood.2, 2024-08-09
* ...
* Sumac.1, 2024-12-09
* ...

**Comment by antoviaque** [comment]
Adolfo: Reorganize milestones to target specific milestones would allow to replace the spreadsheet that . A ticket can't be in two milestones at a time, and has to set a date, so it's better than feature-based releases. It targets point releases, because we usually target a release at first, but at the last minute it becomes useful to separate tickets between point releases.

Action item: Adolfo will post a more complete proposal.

**Comment by antoviaque** [comment]
@arbrandes Did you get a chance to look into this? Or maybe this is currently blocked?

**Comment by antoviaque** [comment]
@arbrandes I think we can call this blocked now no? :) Or did you get to make progress on it?

If it's not going to happen soon, we could also move it back to the backlog, this way I won't be pestering you ;p

**Comment by antoviaque** [comment]
@arbrandes I'll move this task to the backlog for now - don't hesitate to move it back to in progress when you are able to work on it.

**Comment by arbrandes** [comment]
Yeah, sorry I haven't pushed this forward, yet.  I'll try and make some time for it before Teak.  I'll change the status once I write the proposal.
